src/region/RegionManager.tsx

```diff
@@ -35,6 +36,7 @@ export default class RegionManager implements BaseManager {
         const rootLayerEl = documentEl.createElement('div');
         rootLayerEl.classList.add('ba-Layer');
         rootLayerEl.classList.add('ba-Layer--region');
+        rootLayerEl.style.cursor = `url('data:image/svg+xml;utf8,${ICON_REGION_CROSSHAIR}'), crosshair`;
```

Comment:
We would only want to show the cursor in "create" mode, right? And stop showing it once the user types a message in the popup?

Reply:
> We would only want to show the cursor in "create" mode, right? And stop showing it once the user types a message in the popup?

oh, I thought that is only for tooltip to disappear after popup shows up. I will confirm it with Matt

Reply:
> > We would only want to show the cursor in "create" mode, right? And stop showing it once the user types a message in the popup?
> 
> oh, I thought that is only for tooltip to disappear after popup shows up. I will confirm it with Matt

Confirmed. "we only show it while the user can create a box". yeah, I'll change the implementation

Reply:
Cool. Can we also find a way to do it via a CSS class rather than inline styles? That way platform customers could customize the cursor if they wanted.

Reply:
> Cool. Can we also find a way to do it via a CSS class rather than inline styles? That way platform customers could customize the cursor if they wanted.

yeah, discussed on slack